input-output-hk/cardano-sl
# [CSE-246] Un-/Subscription to last page of epochs

_Task_: https://issues.serokell.io/issue/CSE-246

_How to test:_
1. [Build and run](https://github.com/input-output-hk/cardano-sl/tree/master/explorer/frontend#short-version-of-installation) `Explorer` locally
2. Run `nodes` with `tmux`: `scripts/launch/explorer-with-nodes.sh`
3. Open http://localhost:3100/ in your browser and click to any epoch at `latest blocks` table to open an epoch page. Check adding of new blocks there.
